library/src/main/java/net/grandcentrix/tray/core/Preferences.java

```diff
@@ -301,4 +302,11 @@ static boolean isDataTypeSupported(final Object data) {
                 || data instanceof Boolean
                 || data == null;
     }
+
+    boolean putData(String key, Object value) {
```

Comment:
@StefMa I was thinking it will be reusable by subclasses in the package. I can make it private if it makes more sense that way.